src/styles/navbar/navbar.css

```diff
@@ -1,76 +1,309 @@
-@import url('https://fonts.googleapis.com/css2?family=Oxygen&display=swap');
-
-.side-nav {
-  width: 100%;
-  height: 100vh;
-  background: #3a59fd;
-  font-family: 'Oxygen', sans-serif;
-  color: white;
-  text-align: left;
-}
-
-.side-nav a {
-  color: white;
-}
-
-.side-nav a:hover {
-  color: white;
-  text-decoration: none;
-}
-
-ul {
-  list-style: none;
-  margin-left: -20px;
-}
-
-li {
-  cursor: pointer;
-  font-size: 12px;
-  font-weight: bold;
-}
-
-.user-image {
-  padding-top: 20px;
-}
-
-.user-image img {
-  width: 40px;
-}
-
-.welcome {
-  padding-top: 20px;
-}
-
-.username {
-  font-weight: bold;
-  line-height: 2;
-}
-
-.side-routes {
-  padding-top: 50px;
-}
-
-.route {
-  padding-top: 25px;
-}
-
-.icon {
-  background: #497bfe;
-  width: 25px;
-  padding: 6px;
-  border-radius: 5px;
-}
-
-.routes {
-  display: flex;
-}
-
-.name {
-  padding-left: 10px;
-  padding-top: 2px;
-}
-
-.logout {
-  position: absolute;
-  bottom: 50px;
-}
+/*--------------------------------------------------------------
+	## Header
+--------------------------------------------------------------*/
+.header-area {
+  padding: 0 0 98px;
+  position: relative; }
+  .header-area .banner-slider-item {
+    height: 96vh; }
+  .header-area .header-inner-details {
+    height: 100vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center; }
+  .header-area .banner-thumb-wrap {
+    height: 100vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center; }
+  .header-area .sub-title {
+    font-size: 20px;
+    line-height: 52px;
+    color: #fff;
+    margin-bottom: 5px;
+    font-weight: 100;
+    font-family: var(--poppins-font) !important; }
+    .header-area .sub-title.style-two {
+      color: var(--main-color-two); }
+  .header-area .title {
+    font-size: 65px;
+    line-height: 74px;
+    color: #01358D;
+    font-weight: 700;
+    margin-bottom: 30px;
+    font-family: var(--poppins-font) !important; }
+    .header-area .title span {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)